src/web/libdoc/i18n/translations.json

```diff
@@ -56,5 +56,34 @@
     "generatedBy": "Luotu",
     "on": "",
     "chooseLanguage": "Valitse kieli"
+  },
+  "nl": {
+    "code": "nl",
+    "intro": "Introductie",
+    "libVersion": "Bibliotheek versie",
+    "libScope": "Bibliotheek bereik",
+    "importing": "Importeren",
+    "arguments": "Parameters",
+    "doc": "Documentatie",
+    "keywords": "Actiewoorden",
+    "tags": "Labels",
+    "returnType": "Andwoord type",
+    "kwLink": "Actiewoord link",
+    "argName": "Benoemde parameters",
+    "varArgs": "Variabel aantal parameters",
+    "varNamedArgs": "Variable aantal benoemde parameters",
+    "namedOnlyArg": "Alleen benoemde parameters",
```

Comment:
Ik denk dat _alleen_ niet de juiste term is hier. Dit zijn de parameters die niet anders dan als benoemde parameters gebruikt kunnen worden. Misschien dat _verplicht benoemde parameters_ beter is? _Parameters die altijd een naam (moeten) hebben_?

Of, in plaats van _benoemde parameters_, _parameters met naam_. En dan wordt deze _parameters met altijd een naam_. Dat is net iets minder lang. Het alternatief _altijd benoemde parameters_ is namelijk niet waar. Ze worden niet altijd benoemd, maar als ze benoemd worden moeten ze altijd een naam hebben. Best een lastige, deze.